dnadoh/synthesize.py

```diff
@@ -266,10 +276,8 @@ def _write_reference_genome(options, genomes):
 
 def _write_variants(options, genomes, people):
     """Write one variant file per person."""
-    width = max(2, len(str(len(people))))
-    for (pid, person) in enumerate(people):
-        pid_str = str(pid + 1).zfill(width)
-        filename = f"{options.output_stem}-pid{pid_str}.csv"
+    for person in people:
+        filename = f"{options.output_stem}-pid{person.pid}.csv"
         with open(filename, "w") as raw:
             writer = csv.DictWriter(raw, fieldnames=["loc", "base"])
```

Comment:
```suggestion
            writer = csv.DictWriter(raw, fieldnames=["location", "base"])
```

change loc --> location because it's easier to understand and avoid  overlap with `loc` in pandas

Reply:
this change was pushed to #8 so this PR LGTM now